tests/test_merge.py

```diff
@@ -51,33 +51,261 @@ def test_merge_branch(wst, capsys):
     assert out.split('\n')[0] == 'Merging 3.0.x into master'
 
 
-def test_merge_branch_with_whitelist(wst, capsys):
+def test_merge_branch_skip_last(wst, capsys):
     config.merge.branches = '1.0.x 2.0.x 3.0.x master'
+    with temp_git_repo():
+        # Dummy commit
+        run('git commit --allow-empty -m dummy-commit')
+
+        run('git checkout -b 3.0.x')
+
+        # commit 1
+        run('touch temp1.xml')
+        run('git add -A')
+        run('git commit -m commit1')
+
+        # commit 2
+        run('touch temp2.xml')
+        run('git add -A')
+        run('git commit -m  commit2')
+
+        # commit 3
+        run('touch temp3.xml')
+        run('git add -A')
+        run('git commit -m  [skip]')
+
+        run('git checkout master')
+
+        wst('merge 3.0.x --skip-commits \'skip\' \'space separated example\'')
 
+        changes = run('git log --oneline', return_output=True)
+        # Change should have been in the log entry
+        #assert 'Merge branch \'3.0.x\'' in changes
+        #assert '[skip]' in changes
+        #assert 'commit2' in changes
+
+        from os import listdir, popen
```

Comment:
move the import to the top of the file so you don't have to repeat it in every method